www/index.html

```diff
@@ -39,7 +46,7 @@
     <script src="lib/moment/min/locales.min.js"></script>
     <script src="lib/moment/min/moment-with-locales.min.js"></script>
     <script src="lib/moment-timezone/builds/moment-timezone-with-data.min.js"></script>
-    <script src="lib/ngCordova/dist/ng-cordova.js"></script>
+    <!-- <script src="lib/ngCordova/dist/ng-cordova.js"></script> -->
```

Comment:
Note to self: we removed the ngCordova library as part of 
https://github.com/e-mission/e-mission-phone/commit/61513d92e2223aabef929c072b05110b518c0462#diff-dedb6c21fdaac9cba4da4afde881ffe5c80f80896274dbf7971d312fc6f5207d
since it is now deprecated.

Reply:
Right, that is why it is a note **to self**